src/main/java/uk/gov/justice/digital/repository/DomainRepository.java

```diff
@@ -1,29 +1,27 @@
 package uk.gov.justice.digital.repository;
 
 import uk.gov.justice.digital.client.dynamodb.DynamoDBClient;
-import uk.gov.justice.digital.common.Util;
-import uk.gov.justice.digital.domains.model.DomainDefinition;
-import uk.gov.justice.digital.service.DeltaLakeService;
-import java.util.HashSet;
-import java.util.Set;
+import uk.gov.justice.digital.domain.model.DomainDefinition;
 
-public class DomainRepository extends Util {
 
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.PatternSyntaxException;
 
+public class DomainRepository {
 
     private final DynamoDBClient dynamoDBClient;
 
-    protected DeltaLakeService service = new DeltaLakeService();
-
     public DomainRepository(final DynamoDBClient dynamoDBClient) {
```

Comment:
This should be annotated `@Inject` in order to have Micronaut pass in an instance of the client.